src/style.css

```diff
@@ -0,0 +1,69 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  width: 96px;
+}
+
+.stars--0,
```

Comment:
As mentioned in the html structure, If you would give the containing divs also `stars` class then you wouldn't need to list all of those classes here.